api/pyproject.toml

```
@@ -1,28 +1,39 @@
-[tool.poetry]
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/"]
+
+[project]
 name = "api"
 version = "1.4.0" # x-release-please-version
 description = "API for Template Fastapi React"
 authors = []
-license = ""
-
-[tool.poetry.dependencies]
-cachetools = "^5.3.0"
-python = "^3.10"
-fastapi = "^0.101.0"
-pyjwt = "^2.8.0"
-uvicorn = {extras = ["standard"], version = "^0.21.1"}
-pymongo = "4.1.1"
-certifi = "^2023.7.22"
-httpx = "^0.23.3"
-opencensus-ext-azure = "^1.1.9"
-pydantic = "^2.1"
-pydantic-settings = "^2.0.1"
-pydantic-extra-types = "^2.0.0"
-
-[tool.poetry.dev-dependencies]
-pre-commit = ">=3"
-pytest = "^7.2.2"
-mongomock = "^4.1.2"
+requires-python = ">=3.10"
```

```suggestion
requires-python = ">=3.10"
```
Should probably refine this, docker uses 3.11 now, while locally/pre-commit using uv will select latest supported python version -> 3.12